blockchain/fullblocktests/generate.go

```diff
@@ -726,42 +729,42 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 
 	// Create a block that generates too much proof-of-work coinbase.
 	//
-	//   ... -> b1(0) -> b2(1) -> b5(2) -> b6(3)
-	//                                         \-> b9(4)
-	//               \-> b3(1) -> b4(2)
-	g.SetTip("b6")
-	g.NextBlock("b9", outs[4], ticketOuts[4], additionalCoinbasePoW(1))
+	//   ... -> bf1(0) -> bf2(1) -> bf5(2) -> bf6(3)
+	//                                             \-> bpw1(4)
```

Comment:
The arrows on the next line should start after the parens like so:

```Go
//   ... -> bf1(0) -> bf2(1) -> bf5(2) -> bf6(3)
//                                              \-> bpw1(4)
```

The reason is because then it can be copied and extended like so with the arrows still lining up:

```Go
//   ... -> bf1(0) -> bf2(1) -> bf5(2) -> bf6(3) -> somethingelse....
//                                              \-> bpw1(4)
```